rs/nns/governance/src/governance.rs

```diff
@@ -275,6 +275,54 @@ impl NetworkEconomics {
             voting_power_economics: Some(VotingPowerEconomics::with_default_values()),
         }
     }
+
+    /// Returns a modified copy of self where fields containing the default
+    /// value are replaced with the value from defaults. In particular, 0 and
+    /// None are replaced.
+    fn inherit_from(&self, defaults: &Self) -> Self {
+        /// Returns ours if it is nonzero. Otherwise, returns default.
+        fn inherit_from<Primitive>(ours: Primitive, default: Primitive) -> Primitive
+        where
+            Primitive:
+                Default
+                + Eq
+                // Not actually used. This is just to make sure we only support numbers.
+                // This could be relaxed (i.e. deleted) later.
+                + std::ops::Add,
+        {
+            if ours == Primitive::default() {
+                return default;
+            }
+
+            ours
+        }
+
+        Self {
```

Comment:
This is something that Andre suggested in order to make sure that future changes to NetworkEconomics is always supported by ManageNetworkEconomics.